odl/solvers/advanced/proximal_operators.py

```diff
@@ -446,6 +436,33 @@ def proximal_box_constraint(space, lower=None, upper=None):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    If :math:`P` is an interval :math:`[a,b]`, the indicator function is
+    defined as
+
+        .. math::
+
+            I_{x \\in P} = \\left\{ \\begin{array}{ll}
```

Comment:
Specific reason not to use `\begin{cases} \end{cases}`? It doesn't require the `\left \right` construction and sets appropriate horizontal spaces automatically.